src/components/ContactInfo/ContactInfo.vue

```diff
@@ -1,120 +1,118 @@
-          <span v-if="!!businessContact.extension">Ext: {{ businessContact.extension }}</span>
-        </div>
-        <div id="lbl-no-phone" v-else class="info-text">(Not entered)</div>
-      </v-col>
-
-      <!-- Edit Actions -->
-      <v-col v-if="!disableActions" cols="1" class="mt-n2">
-        <div class="actions mr-4">
-          <v-btn
-            v-if="hasBusinessContactInfoChange"
-            id="btn-undo-contact-info"
-            text color="primary"
-            class="undo-action"
-            @click="resetContactInfo()"
-          >
-            <v-icon small>mdi-undo</v-icon>
-            <span>Undo</span>
-          </v-btn>
+    <!-- SUMMARY SECTION -->
+    <template v-if="!isEditing">
+      <!-- Display Contact Info -->
+      <v-row no-gutters class="edit-section">
+        <v-col cols="12" sm="3" class="pr-4">
+          <label class="contact-info-label">
+            <span>{{ contactLabel }} Contact Information</span>
+            <v-flex>
+              <v-chip v-if="hasBusinessContactInfoChange" x-small label color="primary" text-color="white">
+                {{editedLabel}}
+              </v-chip>
+            </v-flex>
+          </label>
+        </v-col>
 
-          <v-tooltip
-            v-else top
-            content-class="top-tooltip"
-            transition="fade-transition"
-            nudge-right="3"
-            :disabled="disableActionTooltip"
-          >
-            <template v-slot:activator="{ on }">
-              <v-btn
-                v-on="on"
-                id="btn-correct-contact-info"
-                text color="primary"
-                @click="isEditing = true"
-              >
-                <v-icon small>mdi-pencil</v-icon>
-                <span>{{editLabel}}</span>
-              </v-btn>
-            </template>
-            <span>No fee to change</span>
-          </v-tooltip>
-
-          <!-- Drop Down Actions -->
-          <span class="more-actions" v-if="hasBusinessContactInfoChange">
-            <v-menu
-              offset-y left nudge-bottom="4"
-              v-model="dropdown"
+        <!-- Email Address -->
+        <v-col cols="12" sm="4" class="pr-4">
+          <label class="item-label">Email Address</label>
+          <div id="lbl-email">
+            {{ !!businessContact.email ? businessContact.email : "(Not entered)" }}
+          </div>
+        </v-col>
+
+        <!-- Phone Number -->
+        <v-col cols="6" sm="3" class="pr-4">
+          <label class="item-label">Phone Number</label>
+          <div v-if="!!businessContact.phone" id="lbl-phone">
+            {{ businessContact.phone }}
+            <span v-if="!!businessContact.extension">Ext: {{ businessContact.extension }}</span>
+          </div>
+          <div v-else id="lbl-no-phone">(Not entered)</div>
+        </v-col>
+
+        <!-- Edit Actions -->
+        <v-col v-if="!disableActions" cols="6" sm="2">
+          <div class="d-flex justify-end align-end align-sm-start">
```

Comment:
Flex container for the button, which aligns it to the bottom by default, and the top from size sm and greater.